spec/classes/pulpcore_spec.rb

```diff
@@ -14,11 +14,30 @@
 
         it do
           is_expected.to compile.with_all_deps
+          is_expected.to contain_class('postgresql::server')
```

Comment:
You can also move this to the base context (line 10) to keep this test just about adding a plugin.